Fundamentals-2-Ind.md

```diff
@@ -88,7 +88,7 @@ Documentation to declare included free open source software components as either
 
 Suggested formats are being developed both for general consumption and for industry niches like automotive, health and telecom. There is a standard for the data format (SPDX) for this in place. ISO/IEC 5962:2021
 
-[Link to NTIA requirements on SBOMs >>](https://www.ntia.doc.gov/report/2021/minimum-elements-software-bill-materials-sbom)
+[Link to NTIA requirements on SBOMs >>](https://www.ntia.doc.gov/report/2021/minimum-elements-software-bill-materials-sbom){:target="_blank"}
```

Comment:
Very good keeping the original page open while allowing readers to explore linked page outside.